.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,51 @@
+name: E2E Tests
+on:
+  workflow_run:
+    workflows:
+      - "Vercel Deployment"
+      - "Vercel Deployment erd-sample"
+    types:
+      - completed
+
+env:
+  CI: true
+  WORKING_DIRECTORY: .
```

Comment:
Is it necessary?

Reply:
Thank you.🙏 I've removed env section.
https://github.com/liam-hq/liam/pull/660/commits/a9e47e3099e4ff25168e070cfc04b7d6162c7d12